src/command.c

```diff
@@ -18,13 +18,14 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+#include <glib/gstdio.h>
```

Comment:
This is an unrelated change, please don't move the header as part of this patch.